src/lib/storage-utils.ts

```diff
@@ -188,13 +220,14 @@ export function mergeUplinkTimeIntoLocal(localMetadata: Package, remoteMetadata:
   return localMetadata.time;
 }
 
-export function prepareSearchPackage(data: Package, time: mixed) {
-  const listVersions: Array<string> = Object.keys(data.versions);
-  const versions: Array<string> = semverSort(listVersions);
-  const latest: string = data[DIST_TAGS] && data[DIST_TAGS].latest ? data[DIST_TAGS].latest : versions.pop();
+export function prepareSearchPackage(data: Package, time: unknown): any {
+  const listVersions: string[] = Object.keys(data.versions);
+  const versions: string[] = semverSort(listVersions);
+  const latest: string | undefined = data[DIST_TAGS] && data[DIST_TAGS].latest ? data[DIST_TAGS].latest : versions.pop();
 
-  if (data.versions[latest]) {
+  if (latest && data.versions[latest]) {
     const version: Version = data.versions[latest];
+    const versions: any = { [latest]: 'latest' };
     const pkg: any = {
```

Comment:
how different is this from `Package` type?

Reply:
Not 100% sure, this code is being used by `npm search` command. But that's something might change soon https://github.com/verdaccio/verdaccio/issues/310 so I would put it in the bottom of the priority queue.